### ISSUE TITLE
[RFC] Grid sorting - default order

### ISSUE THREAD

**Issue by NeverResponse** [Sylius/Sylius#6227]
The current implementation takes in account the sorting order only of the first stated field.
Doesn't matter how many other fields are declared; their order is always descending.

``` yaml
sorting:
    code:
        path: code
        direction: asc #first gets sorted as declared
    name:
        path: translation.name
        direction: asc #ignored altogether and defaulted to desc
```

So I'd like to ask, what would be the preferred default behaviour for sorting with multiple fields?

So far came up with two ways of handling the issue, but if anyone would had something different in mind please share:

**1. Each field having it's own sorting order**
So it's basically a patch to the current code base with acknowledgement of other fields' direction parameter.

**2. Changing the configuration to have a global sorting order for all fields in the model**
Something alike:

``` yaml
sorting:
    direction: asc
    code:
        path: code
    name:
        path: translation.name
```

While I'm in favour of the first one, cause it gives more flexibility, but the fact that you'd get different results after clicking headers sets me off a bit. Seems it might create a lot of confusion.

**Unless** the UI gets changed so it'll show the default order for each sortable column.

Taking any suggestions~
Cheers

**Comment by pamil** [comment]
Isn't it solved by #6439?

**Comment by vasilvestre** [comment]
I've tried and it doesn't look solved in 1.4.12. Maybe I could create a new issue ?
